pkg/gatherers/conditional/gathering_functions.go

```diff
@@ -78,6 +78,8 @@ type GatherLogsOfNamespaceParams struct {
 	Namespace string `json:"namespace"`
 	// A number of log lines to keep for each container
 	TailLines int64 `json:"tail_lines"`
+	// Container name to specify the containers to get log from
+	Container string `json:"container,omitempty"`
```

Comment:
Is it one container name or would it make sense to have more?

Reply:
It supports Regex. So, we are good with just one.